text/0006-monolothic-packaging.md

```diff
@@ -279,6 +279,37 @@ Open issues:
 - Should we use `aws_cdk` as the module name to preserve compatibility or rename
   to `aws_cdk_lib`?
 
+### Go
+
+Usage:
+
+```go
+import (
+    "github.com/aws/aws-cdk-go/awscdk"
+)
+
+app := awscdk.NewApp()
+stack := awscdk.NewStack(app, "MyStack");
+bucket := awscdk.awss3.NewBucket(stack, "MyBucket")
```

Comment:
Are you sure `awscdk.awss3` is a thing? If `awss3` is a "nested" package, I believe the nesting has to be embodied in the import path, and such dot-wise package traversal isn't possible. 

It would have to be:

```go
import (
    "github.com/aws/aws-cdk-go/awscdk"
    "github.com/aws/aws-cdk-go/awscdk/awss3"
)

app := awscdk.NewApp()
stack := awscdk.NewStack(app, "MyStack");
bucket := awss3.NewBucket(stack, "MyBucket")
``` 

Which is a little awkward because now `awscdk` actually only refers to the core objects.

So maybe:

```go
import (
    "github.com/aws/aws-cdk-go/core"
    "github.com/aws/aws-cdk-go/awss3"
)

app := core.NewApp()
stack := core.NewStack(app, "MyStack");
bucket := awss3.NewBucket(stack, "MyBucket")
```

Where each import path represents a top level directory in the repo. Users can still always alias to whatever they want.

Reply:
You are right. Fixed.